products-service/src/functions/getAllProducts/handler.ts

```diff
@@ -0,0 +1,15 @@
+import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/api-gateway';
+import { formatJSONResponse } from '@libs/api-gateway';
+import { middyfy } from '@libs/lambda';
+import { products } from 'src/mockData';
+
+import schema from './schema';
+
+const getAllProducts: ValidatedEventAPIGatewayProxyEvent<typeof schema> = async (event) => {
+  return {
+    statusCode: 200,
+    body: JSON.stringify(products)
+  };
```

Comment:
Might add headers, because your app doesn't show content.
![image](https://user-images.githubusercontent.com/73526360/222142043-eacc3393-10eb-4e72-8334-bb7c88fbdc74.png)

Reply:
Strange.... I'll check it

Reply:
@kayarmolenka Done! Please check